PKGBUILD

```diff
@@ -25,6 +26,12 @@ sha256sums=('986125b92192e404a2f3af5db510d2d651c6301d218cbb66edd6013f8e8153b0'
 build() {
   cd "server-$pkgver"
   local build_date=$(date "+%F-%T" -d "@${SOURCE_DATE_EPOCH}")
```

Comment:
Move this before the `go build` command.

Reply:
What exactly do you mean with
> Do you want to adjust check() and install() in this PR as well or create a new one?

Install doesn't need any changes, the static files are included in the binary. What should be changed in check? Like testing that the ui *.html files doesn't return 404?

Reply:
Ok, interesting.

I also thought about running the yarn tests, but I guess they can't run headless?